provisioning/miniprov/hare_mp/main.py

```diff
@@ -42,7 +42,7 @@
 
 # Logger details
 LOG_DIR = "/var/log/seagate/hare/"
-LOG_FILE = 'setup.log'
+LOG_FILE = "/var/log/seagate/hare/setup.log"
```

Comment:
@supriyachavan4398, I remember I had commented to use `os.path.join(LOG_DIR, LOG_FILE)` and you had made the change, was that overwritten somehow?

Reply:
> can we make use of LOG_DIR variable instead of path? OR if we do not need LOG_DIR variable, shall we remove it? @supriyachavan4398 @SwapnilGaonkar7

Yes we can remove it. but I used this variable in create_logger_directory() function.

Reply:
Ok. then may be @mssawant's comment needs to be addressed. do not remove the variable if its used at other place.

Reply:
> @supriyachavan4398, I remember I had commented to use `os.path.join(LOG_DIR, LOG_FILE)` and you had made the change, was that overwritten somehow?

This changes suggested by Konstantin. For reference I added conversation into it. [https://github.com/Seagate/cortx-hare/pull/1630#discussion_r647502667](url)

Reply:
Post discussion in the DSU, the `os.path.join` was reverted which was not replaced by the absolute path of the setup.log file.
Having LOG_DIR would help in case of more than 1 log files. Anyway, let's complete this patch anyway.

Reply:
@mssawant , I used this variable in create_logger_directory() function.

Reply:
@mssawant , I used this variable in create_logger_directory() function.

Reply:
The path is redundant in LOG_DIR variable and in LOG_FILE. This needs to be normalized. Let's complete this for now.